### DISCUSSION TITLE
Use of left modifiers exclusively

### DISCUSSION THREAD

**Discussion by aadcg** [manna-harbour/miryoku#221] [Q&A] [answered]
I'm wondering why @manna-harbour decided to use left modifers for both the right and left home rows. I couldn't any information about it. Thanks.

**Comment by manna-harbour** [comment, marked as answer]
In some cases the OS and applications distinguish between left and right mods. But in Miryoku mods should be able to be used equally with either hand, so they are all the same. And in the case that you need a right mod for a different purpose, they would need to be available on both hands for the same reason. 

Typically it's only right Alt that is distinguished, when used as AltGr, so AltGr is available below Alt on both sides.

Also see https://github.com/manna-harbour/miryoku/issues/33, https://github.com/manna-harbour/miryoku/discussions/53, https://github.com/manna-harbour/miryoku/discussions/22.

**Comment by ZaxonXP** [comment]
I can add to this. 
I use left alts for both home rows as it allows me to use shortcuts.
I had to move layer taps from both thumbs and remap them to use right alts as I need them for national characters like ęąśżźćółń. 
So in short, the layout is a good base, but it should be adapted for personal needs. :)

**Reply by aadcg** [reply]
Rozumiem.
